codebuild/bin/install_al2_dependencies.sh

```diff
@@ -16,12 +16,47 @@
 set -eu
 source ./codebuild/bin/s2n_setup_env.sh
 
-if [[ ${DISTRO} != "amazon linux" ]]; then
-    echo "Target AL2, but running on $DISTRO: Nothing to do."
-    exit 0
-fi
+al2023_main(){
+    echo "Insatlling AL2023 packages"
+    common_packages
+    al2023_packages
+    versions
+}
+
+al2_main() {
+    echo "Insatlling AL2 packages"
+    common_packages
+    al2_packages
+    symlink_all_the_things
+
+    case "$S2N_LIBCRYPTO" in
+    "openssl-1.1.1")
```

Comment:
Is this the only one we install because AL2 already has 1.0.2 and we don't run any 3.0 builds on AL2?

Reply:
Correct